nilearn/interfaces/fmriprep/tests/test_load_confounds.py

```diff
@@ -577,3 +579,25 @@ def test_inputs(tmp_path, image_type):
     else:
         conf, _ = load_confounds(files)
     assert len(conf) == 2
+
+
+def test_load_confounds_for_gifti_bug_3817(tmpdir):
```

Comment:
NBD, but not sure it's necessary to start adding the issue numbers to test names and the comment is there as reference

Reply:
Good point.
Will also add a line so that people don't have to go and open the issue to know what this is about.